bundles/org.openhab.automation.pythonscripting/pom.xml

```diff
@@ -0,0 +1,262 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>5.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.automation.pythonscripting</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Automation :: Python Scripting</name>
+
+  <properties>
+    <bnd.importpackage>!sun.misc.*,
+      !sun.reflect.*,
+      !com.sun.management.*,
+      !jdk.internal.reflect.*,
+      !jdk.vm.ci.services</bnd.importpackage>
+
+    <graalpy.version>24.1.2</graalpy.version>
+  </properties>
+
+  <dependencies>
+    <!-- Graal Polyglot Framework -->
+    <dependency>
+      <groupId>org.graalvm.polyglot</groupId>
+      <artifactId>polyglot</artifactId>
+      <version>${graalpy.version}</version>
+    </dependency>
+    <!-- Graal TRegex engine (internally used by Graal Python engine) -->
+    <dependency>
+      <groupId>org.graalvm.regex</groupId>
+      <artifactId>regex</artifactId>
+      <version>${graalpy.version}</version>
+    </dependency>
+    <!-- Graal Python engine (depends on Graal TRegex engine, must be added after it) -->
+    <dependency>
+      <groupId>org.graalvm.polyglot</groupId>
+      <artifactId>python-community</artifactId>
+      <version>${graalpy.version}</version>
+      <type>pom</type>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>embed-dependencies</id>
+            <goals>
+              <goal>unpack-dependencies</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-scm-plugin</artifactId>
+        <version>2.1.0</version>
+        <configuration>
+          <providerImplementations>
+            <git>jgit</git>
+          </providerImplementations>
+        </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.apache.maven.scm</groupId>
+            <artifactId>maven-scm-provider-jgit</artifactId>
+            <version>2.1.0</version>
+          </dependency>
+        </dependencies>
+        <executions>
+          <execution>
+            <id>checkout-openhab-python</id>
+            <goals>
+              <goal>checkout</goal>
+            </goals>
+            <phase>generate-resources</phase>
+            <configuration>
+              <connectionUrl>scm:git:https://github.com/HolgerHees/openhab-python</connectionUrl>
```

Comment:
It would be great to move this repo under the OpenHAB [GH org](https://github.com/openhab), similar to how JRuby helpers are present [here](https://github.com/openhab/openhab-jruby/).
This way not only would the repo attract more maintainers and gain governance, but it'll also allow to version these libs together with OH releases (which I think we should consider to stabilize these interfaces).
**WDYT**?

----
_Side note_: I'm no maintainer nor do I have any authoritative say in the matter, so feel free to disregard this suggestion entirely. Just stumbled across this PR _(awesome work, BTW 🙏 )_ and considered it would be more "secure" if the build did not bundle "external" resources.

Reply:
Hi @mbronk ,

That's the plan. I want to move it after the review, but fore the final release and together with openHAB 5. But for the starting phase, I have not find a better place :-)